rpc/src/lib.rs

```diff
@@ -226,13 +231,18 @@ pub fn start_ipc<M, S, H, T>(
 	addr: &str,
 	handler: H,
 	extractor: T,
+	chmod: u16
 ) -> ::std::io::Result<ipc::Server> where
 	M: jsonrpc_core::Metadata,
 	S: jsonrpc_core::Middleware<M>,
 	H: Into<jsonrpc_core::MetaIoHandler<M, S>>,
 	T: IpcMetaExtractor<M>,
 {
+	let attr = SecurityAttributes::empty()
+		.set_mode(chmod as _)?;
```

Comment:
should not be needed anymore

```suggestion
		.set_mode(chmod)?;
```